### ISSUE TITLE
Display output of commandline

### ISSUE THREAD

**Issue by dotMorten** [AnyStatus/Support#174]
I'd like the ability to run a batch script and display the output in the status. Status is green if the script succeeds, and the description will show the output.
For example this would get the latest build number from a file and display it in the AnyStatus:
```
SET /p buildnum=<\\builderver\currentbuildnumber.txt | echo %buildnum%
```

**Comment by AlonAm** [comment]
At moment you can control the status by returning different exit codes but you can't show a text message. I'll try to provide an option to do so.

**Comment by dotMorten** [comment]
btw would be nice if I could actually just enter the commandline script, rather than having to point to a file. That makes it easier to export and backup or share a configuration and include the actual commands.

**Comment by AlonAm** [comment]
Absolutely. I future versions, I'd like to enable inline scrips, C#, PowerShell and others. As well as syntax highlighting. Something like linqpad. Currently only the SQL Scalar widget enables inline scripts.

**Comment by AlonAm** [comment]
Side note, in AnyStatus 3, the app, user, session and data (widgets) are saved in separate json files. The Add Widget UI is also significantly improved and simplified.

**Comment by AlonAm** [comment]
Also, expected breaking API changes. Overall, v3 will not be backward compatible but it fixes several early design mistakes.
